ee/modules/110-istio/monitoring/prometheus-rules/multicluster.yaml

```diff
@@ -52,3 +52,26 @@
           TOKEN="$(deckhouse-controller helper gen-jwt --private-key-path <(echo "$KEY") --claim iss=d8-istio --claim sub=$LOCAL_CLUSTER_UUID --claim aud=$REMOTE_CLUSTER_UUID --claim scope=private-multicluster --ttl 1h)"
           curl -H "Authorization: Bearer $TOKEN" {{$labels.endpoint}}
           ```
+
+
+    - alert: D8IstioRemoteClusterNotSynced
+      expr: istio_remote_cluster_up == 0
```

Comment:
Make sure if the cluster in HA mode and remote cluster fails. There must be a single ClusterAlert `D8IstioRemoteClusterNotSynced` (in prometheus there is a single metric istio_remote_cluster_up with labels cluster_id and istiod).